### PR TITLE
remove oee files and references

### PR DESCRIPTION
**Pull request by BeateRixen**
Ask requested by @rahultsag the OEE documentation and all references will be removed from the CD, y2024 and future documentation.

### CODE REVIEW

File: content/dtm/asset-types-bundle/asset-models.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ To create a root asset model follow a bottom-up approach by defining all child a
 
 This root asset model can then be used to create an asset hierarchy, which then defines all the assets, subassets, asset properties and devices in the DTM application.
 
-As a result you can use the asset hierarchy in other {{< product-c8y-iot >}} applications, such as the {{< product-c8y-iot >}} OEE or the {{< product-c8y-iot >}} Machine Portal.
+As a result you can use the asset hierarchy in other {{< product-c8y-iot >}} applications, such as the {{< product-c8y-iot >}} Machine Portal.
```
Comment (BeateRixen) [inline content/dtm/asset-types-bundle/asset-models.md]: @pavanshivajirao @rahultsag Please check the phrasing here and below in the next file.

**Comment by BeateRixen** [comment]
@MWindel Please create a PR for the removal of the OEE OPS documentation in c8y-docs-ops accordingly.

File: content/dtm/asset-types-bundle/asset-models.md (hunk 1 above)
Comment (rahulc8y) [reply content/dtm/asset-types-bundle/asset-models.md]:
```suggestion
As a result you can use the asset hierarchy in other {{< product-c8y-iot >}} applications, such as the {{< product-c8y-iot >}} Cockpit.
```

**Comment by MWindel** [comment]
@BeateRixen All OEE content and references have been removed in the operations documentation. See https://github.com/Cumulocity-IoT/c8y-docs-ops/pull/902.

File: content/dtm/dtm-introduction.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Welcome to the {{< product-c8y-iot >}} Digital twin manager (DTM) application.
 
 This application allows you to create and manage assets around your physical connected devices in {{< product-c8y-iot >}} using a digital representation of a physical hierarchy of assets and devices. Here, assets can be used to structure and describe devices in logical hierarchies as encountered in the real world environment to help with visualizing the hierarchy in a digital setting.
 
-Once the asset hierarchy is created in the DTM application, the hierarchy can be applied to other applications such as OEE or Machine portal. It saves a lot of time and effort, as the hierarchy is created only once and can be used directly in other {{< product-c8y-iot >}} hosted applications.
+Once the asset hierarchy is created in the DTM application, the hierarchy can be applied to other applications such as the {{< product-c8y-iot >}} Machine Portal. It saves a lot of time and effort, as the hierarchy is created only once and can be used directly in other {{< product-c8y-iot >}} hosted applications.
```
Comment (BeateRixen) [inline content/dtm/dtm-introduction.md]:
```suggestion
Once the asset hierarchy is created in the DTM application, the hierarchy can be applied to other applications such as the {{< product-c8y-iot >}} Cockpit. It saves a lot of time and effort, as the hierarchy is created only once and can be used directly in other {{< product-c8y-iot >}} hosted applications.
```

**Review by pavanshivajirao — APPROVED** [review]

**Pull request merged**